examples/create_postcards_from_csv/postcards.py

```diff
@@ -0,0 +1,152 @@
+# usage:
+#   python postcard.py input.csv
+#   logs to csv's in the output directory
+
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../..'))
+
+###########################################################
+# TODO: Provide your API Key, keep this secure
+lob.api_key = 'test_fc26575412e92e22a926bc96c857f375f8b'  # '<your key here>'
+
+# TODO: Create your from_address
+try:
+    from_address = lob.Address.create(
+            name='Your Name',
```

Comment:
Array of dictionary? I like the idea of creating this up here so they know right away if there address or key is bad.